docker-envs/eck/kind-cluster/ror/kbn.yml

```diff
@@ -0,0 +1,61 @@
+apiVersion: kibana.k8s.elastic.co/v1
+kind: Kibana
+metadata:
+  name: quickstart
+spec:
+  version: 8.11.3
+  count: 2
+  elasticsearchRef:
+    name: quickstart
+  config:
+    readonlyrest_kbn.store_sessions_in_index: true
+    readonlyrest_kbn.cookiePass: "12345678901234567890123456789012345678901234567890"
+    readonlyrest_kbn.logLevel: "info"
+  podTemplate:
+    spec:
+      securityContext:
+        runAsUser: 1000
+      containers:
+        - name: kibana
+          volumeMounts:
+            - name: kibana-plugins
+              mountPath: /usr/share/kibana/plugins
+            - name: kibana-node-modules-kbn
+              mountPath: /usr/share/kibana/node_modules/@kbn
+            - name: kibana-src-cli
+              mountPath: /usr/share/kibana/src/cli
+      initContainers:
+        - name: install-ror-kbn-plugin
+          securityContext:
+            allowPrivilegeEscalation: false
+            runAsUser: 0
+          command:
+            - "/bin/bash"
+            - "-c"
+            - |
+              set -e
+              
+              /usr/share/kibana/bin/kibana-plugin install "https://api.beshu.tech/download/kbn?esVersion=$KBN_VERSION&edition=kbn_universal&email=eck-example%40readonlyrest.com"
+              /usr/share/kibana/node/bin/node /usr/share/kibana/plugins/readonlyrestkbn/ror-tools.js patch
+              
+              cp -r /usr/share/kibana/node_modules/@kbn /target/usr/share/kibana/node_modules
+              cp -r /usr/share/kibana/src/cli /target/usr/share/kibana/src
+          env:
+            - name: KBN_VERSION
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.labels['kibana.k8s.elastic.co/version']
+          volumeMounts:
+            - name: kibana-plugins
+              mountPath: /usr/share/kibana/plugins
+            - name: kibana-node-modules-kbn
+              mountPath: /target/usr/share/kibana/node_modules/@kbn
```

Comment:
@Dzuming please confirm that we change files only in these two locations: `node_modules/@kbn` and `src/cli`

Reply:
As I see, we also patch:
- [src/legacy](https://github.com/sscarduzio/readonlyrest_kbn/blob/e4f8c579b75a48a2f17a30fbc57db00895ca38cf/kibana/patchers/patches_for_kbn_distribution/v7.9.x/kbn_server.js.patch)
- [src/core](https://github.com/sscarduzio/readonlyrest_kbn/blob/d48328e1e9df035e36d550adf5bfafe17d4961fb/kibana/patchers/patches_for_kbn_distribution/v8.4.x/server.patch)
- [x-pack/plugins](https://github.com/sscarduzio/readonlyrest_kbn/blob/18802f370ca48505fea184d99bd34ea8e42a2756/kibana/patchers/patches_for_kbn_distribution/v8.5.x/event_log_client.patch)

Reply:
I didnt see these in the patcher logs. I've tested KBN 8.11.

Reply:
We keep poacher files for specific versions here https://github.com/sscarduzio/readonlyrest_kbn/tree/develop/kibana/patchers/patches_for_kbn_distribution

Reply:
If you need patch files only for 8.11.3, the list you provided is correct

Reply:
ok, I've updated the mount paths